mq.tf

```diff
@@ -0,0 +1,159 @@
+variable "mq_apply_immediately" {
+  type        = "string"
+  description = "Specifies whether any cluster modifications are applied immediately, or during the next maintenance window"
+  default     = "true"
+}
+
+variable "mq_enabled" {
+  type        = "string"
+  default     = "true"
+  description = "Set to false to prevent the module from creating any resources"
+}
+
+variable "mq_auto_minor_version_upgrade" {
+  type        = "string"
+  description = "Enables automatic upgrades to new minor versions for brokers, as Apache releases the versions"
+  default     = "false"
+}
+
+variable "mq_deployment_mode" {
+  type        = "string"
+  description = "The deployment mode of the broker. Supported: SINGLE_INSTANCE and ACTIVE_STANDBY_MULTI_AZ"
+  default     = "ACTIVE_STANDBY_MULTI_AZ"
+}
+
+variable "mq_engine_type" {
+  type        = "string"
+  description = "The type of broker engine. Currently, Amazon MQ supports only ActiveMQ"
+  default     = "ActiveMQ"
+}
+
+variable "mq_engine_version" {
+  type        = "string"
+  description = "The version of the broker engine. Currently, Amazon MQ supports only 5.15.0 or 5.15.6."
+  default     = "5.15.0"
```

Comment:
why not `5.15.6`?

Reply:
CodeFresh are using 5.15.0 so we default to that. Only a patch version bump but thought we would start off and a "known working"